client/src/main/java/io/pravega/client/netty/impl/FlowHandler.java

```diff
@@ -194,6 +206,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
             try {
                 processor.process(cmd);
             } catch (Exception e) {
+                log.error("Encountered exception invoking ReplyProcessor.process for reply " + cmd, e);
```

Comment:
Not sure this is needed. The processor should be logging this with more/better context. We'll just end up with duplicated stack traces.

Reply:
updated it.